presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -30,6 +31,8 @@
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.stream.Collectors.toMap;
 
+// TODO: break into operator-specific stats classes instead of having a big union-class aggregating all stats together
+@Deprecated
```

Comment:
This was explicit request from @raghavsethi (https://github.com/prestodb/presto/pull/7661#discussion_r119244354).